forms_pro/api/team.py

```
@@ -77,3 +89,131 @@ def switch_team(team_id: str) -> None:
+    )
+
+
+@frappe.whitelist()
+def add_member_to_team_via_invitation(team_id: str, invite_id: str | None = None) -> None:
+    """
+    Add a member to a team when an invitation is accepted.
+    Accepts invite_id from query param (URL may send it as 'id').
+    """
+    invite_id = invite_id or frappe.form_dict.get("id")
+    if not invite_id:
+        raise frappe.PermissionError("Invitation id is required")
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)